tutorials/tutorial04_meshing/doc/content/getting_started/examples_and_tutorials/tutorial04_meshing/presentation/tutorial04_step12_rgmb_empire.md

```diff
@@ -0,0 +1,117 @@
+# RGMB Example: 2-D Empire Microreactor Core Mesh with Control Drum Stitching
+
+!---
+
+## 2-D Empire Microreactor Core with Control Drum Stitching
+
+This example illustrates the use of RGMB mesh generators to define a heterogeneous 2D hexagonal microreactor core with control drum structures for the EMPIRE problem ([!cite](matthews2021coupled)). The primary focus of this tutorial is to familiarize users with [ControlDrumMeshGenerator.md] to create control drum structures for stitching into the core lattice.
+
+!media tutorial04_meshing/rgmb_empire_stepbystep.png
+       style=width:100%;display:block;margin-left:auto;margin-right:auto;
+
++Hands-on package MOOSE input file (Heterogeneous EMPIRE mesh)+: `combined/reactor_workshop/tests/reactor_examples/rgmb_empire/rgmb_empire.i`
+
+!---
+
+## ReactorMeshParams
+
+[ReactorMeshParams.md] contains global mesh/geometry parameters including whether the final mesh is 2D or 3D, Cartesian or hexagonal, assembly pitch, and the axial discretization for the final extruded geometry. Here we also invoke the option to enable flexible assembly stitching with [!param](/Mesh/ReactorMeshParams/flexible_assembly_stitching), so that dissimilar assembly structutes can be stitched into the reactor core without any hanging nodes. This parameter needs to be set to true in order to use [ControlDrumMeshGenerator.md].
```

Comment:
```suggestion
[ReactorMeshParams.md] contains global mesh/geometry parameters including whether the final mesh is 2D or 3D, Cartesian or hexagonal, assembly pitch, and the axial discretization for the final extruded geometry. Here we also invoke the option to enable flexible assembly stitching with [!param](/Mesh/ReactorMeshParams/flexible_assembly_stitching), so that dissimilar assembly structures can be stitched into the reactor core without any hanging nodes. This parameter needs to be set to true in order to use [ControlDrumMeshGenerator.md].
```